source/tutorials/configure-selinux/configure-selinux.rst

```diff
@@ -0,0 +1,139 @@
+.. _configure-selinux:
+
+SELinux Configuration and Rules
+###############################
+
+Security-Enhanced Linux (`SELinux <https://source.android.com/security/selinux>`_) is enabled in Android to enforce the Mandatory Access Control for security. SELinux supports two working modes: permissive and enforcing:
+
+* In **permissive** mode, it only audits the operations of all domains and prints the AVC (Access Vector Cache) errors that violate the sepolicy rules, but it never blocks any operations.
+
+* In **enforcing** mode, it not only prints out the AVC errors, but also blocks the operations that violate the sepolicy rules.
+
+It's a good practice to set SELinux in permissive mode at the beginning of the development phase, so that you can easily bring up the system and fix any SELinux errors. After resolving all AVC errors, you can change the SELinux mode to enforcing mode.
+
+SELinux Configuration
+=====================
+
+To change SELinux in build time
+-------------------------------
+
+    The SELinux operation mode is configured using *mixins*. You can edit the '*mixins.spec*' file in the source folder for your lunch target at '*device/intel/project-celadon/<target-name>/mixins.spec*', and rebuilt the image to take effect. Set the '*sepolicy*' key to either **permissive** or **enforcing** to enable the corresponding SELinux mode.
```

Comment:
Suggested rewrite: 
The SELinux operation mode is configured using *mixins*. Edit the '*mixins.spec*' file in the source folder for your launch target at '*device/intel/project-celadon/<target-name>/mixins.spec*' and rebuild the image to take effect. ...